src/components/AuthForms/index.js

```diff
@@ -153,10 +211,24 @@ const SignInForm = ({ openSignIn, setOpenSignIn }) => {
             variant="contained"
             color="primary"
             className="login__button"
+            style={{marginLeft:'36%',marginBottom:'6px'}}
+
           >
             Sign In
           </Button>
         </form>
+        <h3 style={{ textAlign: "center" }}>
```

Comment:
Using inline styling is a bad practice, so create a class like I did for other classes. And then change it in style.js, try following the MUI rules.

```jsx
             variant="contained"
             color="primary"
             className={classes.loginButton}
             style={{marginLeft:'36%',marginBottom:'6px'}}
          >
            Sign In
          </Button>
        </form>
        <h3 style={{ textAlign: "center" }}>
          <span>OR</span>
        </h3>
```